src/components/router/Middleware.brs

```diff
@@ -1,17 +1,24 @@
 ' Has to be done with a delay (using Timer) so Router/RouterOutlet is able to catch the change
+' Redirects to a new URL
+' @param {Object} navigationData
+' @param {String} navigationData.path - new URL
+' @param {Object} navigationData.params - query params
```

Comment:
what about `skipInHistory`?

Reply:
it doesn't exist here - it's always overwritten to true